Formula/lib/libxml2.rb

```diff
@@ -70,29 +71,26 @@ def install
                 .to_formula
     inreplace [bin/"xml2-config", lib/"pkgconfig/libxml-2.0.pc"], icu4c.prefix.realpath, icu4c.opt_prefix
 
-    cd "python" do
-      sdk_include = if OS.mac?
-        sdk = MacOS.sdk_path_if_needed
-        sdk/"usr/include" if sdk
-      else
-        HOMEBREW_PREFIX/"include"
-      end
+    sdk_include = if OS.mac?
+      sdk = MacOS.sdk_path_if_needed
+      sdk/"usr/include" if sdk
+    else
+      HOMEBREW_PREFIX/"include"
+    end
+
+    includes = [include, sdk_include].compact.map do |inc|
+      "'#{inc}',"
+    end.join(" ")
+
+    # We need to insert our include dir first
+    inreplace "python/setup.py", "includes_dir = [",
+                                 "includes_dir = [#{includes}"
 
-      includes = [include, sdk_include].compact.map do |inc|
-        "'#{inc}',"
-      end.join(" ")
-
-      # We need to insert our include dir first
-      inreplace "setup.py", "includes_dir = [",
-                            "includes_dir = [#{includes}"
-
-      # Needed for Python 3.12+.
-      # https://github.com/Homebrew/homebrew-core/pull/154551#issuecomment-1820102786
-      with_env(PYTHONPATH: buildpath/"python") do
-        pythons.each do |python|
-          build_isolation = Language::Python.major_minor_version(python) >= "3.12"
-          system python, "-m", "pip", "install", *std_pip_args(build_isolation:), "."
-        end
+    # Needed for Python 3.12+.
+    # https://github.com/Homebrew/homebrew-core/pull/154551#issuecomment-1820102786
+    with_env(PYTHONPATH: buildpath/"python") do
+      pythons.each do |python|
+        system python, "-m", "pip", "install", *std_pip_args, "./python"
```

Comment:
Is dropping `build_isolation` intended? (Or is `build_isolation: true` the default now?)